website/static/CNAME

```diff
@@ -0,0 +1 @@
+www.fast.design
```

Comment:
it appears we have this same file at the root https://github.com/microsoft/fast-dna/blob/master/CNAME do we need this in both places?

Reply:
removed